public/templates/teacher_sets/detail.html

```diff
@@ -13,13 +13,13 @@ <h3>{{ts.title}}</h3>
         <div id='total_available_copies' align="center" style="font-weight:bold"> {{ts.available_copies}} of 
           {{ts.total_copies}} Available | {{ts.new_holds_count}} sets pending
         </div><p style="white-space: pre"></p>
-        <div data-ng-show="ts.available_copies > 0 && !active_hold && is_available ">
+        <div data-ng-show="ts.is_copies_available && is_available ">
           <a ng-click="doReserve()" class="button secondary">{{ts.availability_string}} - Order now!</a>
           <p>Available Teacher Sets will deliver to your school within 2 weeks. For Teacher Sets that are currently in use by other educators, please allow 60 days or more for delivery. If you need materials right away, contact us at <a href="mailto:help@mylibrarynyc.org">help@mylibrarynyc.org</a>.</p>
         </div>
-        <div data-ng-show="active_hold || !is_available">
+        <div data-ng-show="!ts.is_copies_available && !is_available">
```

Comment:
The requirement from the email thread:
"1) Button is available if there are copies available _and_ button hasn’t already been (successfully) clicked
2) Button is always unavailable if there are zero copies, and also unavailable if button has already been clicked

I think in terms of logic, this is essentially adding the "Button is always unavailable if there are zero copies" on top of the existing logic"

Reply:
@daryanypl I tested this functionality  its working as expected . This code for  'unavailble', If we see the this entire  file 'available' code also  there .

Reply:
So you're saying that is_available is updated when the teacher places a hold?

Reply:
@darya  Yes, For more secure already that field is available in our database do added that condition also. If  not required will remove that condition.